src/driver/aurora-data-api-pg/AuroraDataApiPostgresDriver.ts

```diff
@@ -110,7 +110,7 @@ export class AuroraDataApiPostgresDriver extends PostgresWrapper implements Driv
      * Executes given query.
      */
     protected executeQuery(connection: any, query: string) {
```

Comment:
Can we just drop this entirely?

Does the `postgres` version of this work?

```
    /**
     * Executes given query.
     */
    protected executeQuery(connection: any, query: string) {
        return new Promise((ok, fail) => {
            connection.query(query, (err: any, result: any) => {
                if (err) return fail(err);
                ok(result);
            });
        });
    }
```

Reply:
Postgres version misuses the interface defined in the Connection class which defined second parameter of the query method as:
`parameters?: any[]`
And the code you pasted sends in a function.

It feels dirty and it also breaks the code.